src/app/frontend/commons/commons_module.js

```diff
@@ -0,0 +1,11 @@
+import errorDialogService from './errordialog_service';
+
```

Comment:
I'm a little bit worried to introduce `commons` folder. This is because it is like failing to organize codebase and whenever it is hard to find a place for a thing, it lands in common. As a result, you actually dont know what to expect in this folder.

Do you have any idea on what's the contract for this folder? What to place here? I was thinking that such folder can be for non-functional and application-wide services and UI components. 

.. still not convinced and on the fence ...